controllers/cluster_controller.go

```diff
@@ -615,9 +625,112 @@ func (r *ClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&hwameistoroperatorv1alpha1.Cluster{}).
 		Owns(&appsv1.DaemonSet{}).
 		Owns(&appsv1.Deployment{}).
+		Owns(&batchv1.Job{}).Watches(&source.Kind{Type: &batchv1.Job{}},
+		&handler.EnqueueRequestForObject{},
+		builder.WithPredicates(
```

Comment:
may be missed operations for `ADD` events, this is important when job updates during operator crush time

Reply:
ok，I will add it